config.go

```diff
@@ -122,6 +122,13 @@ type CORSConfig struct {
 	//
 	// Added in v4.2.0.
 	AllowAllOrigins bool
+
+	// AllowOrigins enables CORS and allows the list of origins in the
+	// HTTP request origins when set. Practically speaking, this
+	// results in the HTTP header "AllowOrigins".
```

Comment:
Wrong header name :)

```suggestion
	// results in the HTTP header "Access-Control-Allow-Origin".
```